gratipay/models/participant/email.py

```diff
@@ -106,6 +103,16 @@ def validate_email_verification_request(self, c, email, *packages):
         if not all(email in p.emails for p in packages):
             raise EmailNotOnFile()
 
+        unverified_email = c.one("""
+            SELECT address
```

Comment:
We need to start also thinking about not deleting email addresses, because when we start handling bounces and complaints (#4284) we'll want to retain that info for a given email address even if one participant removes it and another (or the same) adds it again. Hmm ...